x/auth/ante/fee.go

```diff
@@ -0,0 +1,99 @@
+package ante
+
+import (
+	"fmt"
+	sdkante "github.com/cosmos/cosmos-sdk/x/auth/ante"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/e-money/em-ledger/x/buyback"
+)
+
+// This is a fork of the DeductFeeDecorator from the SDK version v0.42.4
+// It deposits the fees differently based on what denomination it is paid with.
+// If the fee is paid in NGM, it is sent to the tradition fee pool and distributed as rewards
+// If the fee is paid with a stablecoin balance, it is sent to the buyback module
```

Comment:
Nice documentation but let's follow Go's standard to start with the referenced struct i.e. _DeductFeeDecorator_ otherwise, IDE(s) underline it with squiggly lines making for an unattractive text section :)

Reply:
Fixed this.